samples/bluetooth/enocean/README.rst

```diff
@@ -22,13 +22,13 @@ The LEDs of the board also respond to button presses from an EnOcean switch.
 Requirements
 ************
 
-* One of the following development boards:
+The sample supports the following development kits:
 
-  * |nRF5340DK|
-  * |nRF52840DK|
-  * |nRF52DK|
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340pdk_nrf5340_cpuapp_and_cpuappns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52840dk_nrf52811, nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf52dk_nrf52810
```

Comment:
Maybe change the order to keep the two nRF52840 and the two nRF52 entries together?